### PR TITLE
refactor: repo changes, exporting and rollup bundling

### PR DESCRIPTION
**Pull request by favna**

### CODE REVIEW

**Review by vladfrangu — DISMISSED** [review]

**Review by vladfrangu — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -16,18 +16,21 @@ A very fast input-validation and transformer library.
 
 ## Description
 
-Ever need the performance of very fast input-validation libraries such as [ts-json-validator](https://www.npmjs.com/package/ts-json-validator) with the powerful API of [zod](https://www.npmjs.com/package/zod)? ShapeShift got you covered!
+Ever need the performance of very fast input-validation libraries such as [`ts-json-validator`] with the powerful API of [`zod`]? ShapeShift got you covered!
```
Comment (kyranet) [inline README.md:19]: We should remove the mention to ts-json-validator, it's extremely fast in comparison to this library.

**Review by kyranet — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (favna) [reply README.md:19]: @vladfrangu I'll let you say yay or nay to this

Comment (vladfrangu) [reply README.md:19]: Honestly we should probably just drop the mention of either libraries and just describe that we're one of the fastest object parsing libraries (excluding JSON ones)

Comment (kyranet) [reply README.md:19]: Bump @favna

Comment (favna) [reply README.md]: done

**Review by vladfrangu — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Pull request merged**